grammar.js

```diff
@@ -854,6 +858,126 @@ module.exports = grammar({
         ),
       ),
 
+    // :h sign
+
+    _sign_name: ($) => choice($.integer_literal, $.identifier),
+
+    _sign_define_arg_text: ($) => seq($._printable, optional($._printable)),
+    _sign_define_argument: ($) =>
+      choice(
+        // FIXME: use filename node when filename is merged
+        key_val_arg('icon', optional($.filename)),
+        key_val_arg('linehl', optional($.hl_group)),
+        key_val_arg('numhl', optional($.hl_group)),
+        key_val_arg('text', optional(alias($._sign_define_arg_text, $.text))),
+        key_val_arg('texthl', optional($.hl_group)),
```

Comment:
There is `culhl` missing here.

Reply:
I don't have culhl in my nvim help, is this a vim specific or nvim-v7 ?

Reply:
Yep found it in the neovim doc on github :+1: